src/SDKs/Reservations/Management.Reservations/Generated/SdkInfo_AzureReservationAPI.cs

```diff
@@ -27,5 +26,14 @@ internal static partial class SdkInfo
               }.AsEnumerable();
           }
       }
-  }
+      // BEGIN: Code Generation Metadata Section
+      public static readonly String AutoRestVersion = "latest";
+      public static readonly String AutoRestBootStrapperVersion = "(empty)";
```

Comment:
Was this manually modified?

Reply:
I think this is because I was using the generate.ps1 but didn't update the AutoRestVersion. I will re-generate the file.

Reply:
This is auto-generated.

Reply:
Thanks for confirming

Reply:
Sure. Please feel free to let me know if there is anything else I need to do.